src/frontends/pytorch/src/op/cumsum.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/cum_sum.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_cumsum(NodeContext& context) {
+    // aten::cumsum(Tensor self, int dim, *, ScalarType? dtype=None, Tensor out=None)
```

Comment:
Please also add check that `out` input is None, or try to support the case when `out` is not None.

Reply:
added